### ISSUE TITLE
Waves should spawn after a set amount of time instead of when

### ISSUE THREAD

**Issue by jordan-octavia-blue** [jordan-octavia-blue/Spellmasons#506]
...when the entire current wave is dead because players can just kill time to reset their cooldowns.

https://discord.com/channels/1032294536640200766/1032294537235812404/1208731615266607116

**Comment by jdoleary** [comment]
This feels like an important end-game difficulty adjustment.   Maybe every 5 turns we spawn a new wave and maybe the subsequent waves are smaller.  So the initial spawn will be like 20 units and every wave is an additional 10 or something

**Comment by SoulMuncher** [comment]
I'm cautious of this idea. It's sounds difficult to keep track of, and I'm not sure it totally solves the cooldown issue. If players are ahead of the timer, they can still stall, and if they are behind the timer, enemies spawn on their head mid-combat, which seems kind've frustrating and unpredictable. I'm also wondering if it creates enough of an endgame difficulty spike for all playstyles.

What if we used some other ramping threat or timer to encourage players to clear levels faster overall?
I like the idea of a fatigue modifier that deals increasing damage each turn. Would be automatically applied in the plus levels and can't be purified/contaminated/etc.

**Comment by SoulMuncher** [comment]
> jordan 05/22/2024 3:20 PM
> Maybe we hold off on this for a bit, I've heard players say the + waves are already hard

**Comment by jdoleary** [comment]
> What if we used some other ramping threat or timer to encourage players to clear levels faster overall?
I like the idea of a fatigue modifier that deals increasing damage each turn.

I like this idea, like global threats that aren't attached to enemies.  Maybe each plus level has a unique debuff?  Like Timemasons debuff or what you mentioned.  Thoughts?

**Comment by SoulMuncher** [comment]
Yeah! These could also work as runes. Maybe you choose 1 of a few bad ones at the start of each plus level. Like the old calamity system, but more integrated and interesting
